src/net/mod.rs

```diff
@@ -2070,11 +2070,11 @@ pub mod test {
                     }
                 }
 
-                tx.process_block_txs(&tip, &block_header, &self.config.burnchain, blockstack_ops).unwrap();
+                tx.process_block_txs(&tip, &block_header, &self.config.burnchain, blockstack_ops, None, PoxId::stubbed()).unwrap();
```

Comment:
Is this subbed PoxId desired here?

Reply:
Yeah -- these test cases should be preserved as they are for now. The TestPeer, TestMiner, TestNode, etc. all depend pretty closely on being able to immediately invoke a block processing, and I'd rather preserve these tests behavior than try to rewrite all of these test modules to instead invoke a ChainsCoordinator (which would be tested by either integration tests, or new tests for the Coordinator specifically).